.github/workflows/test.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   build_and_test:
     name: Build and Test
     runs-on: ubuntu-latest
+    environment: Production
```

Comment:
[Q] why environment is production for test workflow?

Reply:
Was testing something, will delete this 😅